Add example on the use of dcterms:conformsTo with dcat:CatalogRecord

andrea-perego opened the pull request:
Summary of changes:
- Added example and introductory text at the end of [§14.2.1 Conformance to a standard](https://raw.githack.com/w3c/dxwg/dcat-ex-record-schema/dcat/index.html#quality-conformance-statement) to show how to use `dcterms:conformsTo` with a `dcat:CatalogRecord` - as per resolution in https://www.w3.org/2022/03/22-dxwgdcat-minutes#r02
- Added Turtle, RDF/XML, and JSON-LD versions of the example in the `dcat/examples/` folder
- Added cross-reference to [§14.2.1 Conformance to a standard](https://raw.githack.com/w3c/dxwg/dcat-ex-record-schema/dcat/index.html#quality-conformance-statement) for guidance on the use `dcterms:conformsTo`
- Replaced "item" with "resource" when referring to an instance of `dcat:Resource` - see https://github.com/w3c/dxwg/issues/1490
- Replaced "item" with "record" when referring to an instance of `dcat:CatalogRecord` - https://github.com/w3c/dxwg/issues/1490
- Updated changelog
- Run US spellchecker
- Editorial fixes

Preview: https://raw.githack.com/w3c/dxwg/dcat-ex-record-schema/dcat/index.html#ex-catalog-record-schema

Diff: https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fw3c.github.io%2Fdxwg%2Fdcat%2F&doc2=https%3A%2F%2Fraw.githack.com%2Fw3c%2Fdxwg%2Fdcat-ex-record-schema%2Fdcat%2Findex.html

andrea-perego committed:
Add example on the use of dct:conformsTo with dcat:CatalogRecord

Implementing resolution in https://www.w3.org/2022/03/22-dxwgdcat-minutes#r02

andrea-perego committed:
Editorial fix

andrea-perego committed:
Add ref to guidance section on use of dct:conformsTo

riccardoAlbertoni commented on dcat/index.html:
```diff
@@ -5472,11 +5479,29 @@ <h3>Conformance to a standard</h3>
+  a dcat:CatalogRecord ;
+  foaf:primaryTopic :dataset-001 ;
+  dcterms:issued "2011-12-11"^^xsd:date ;
+  dcterms:conformsTo &lt;https://www.w3.org.org/TR/vocab-dcat/&gt; ;
```
I would remove teh duplication of .org in the dcterms:conformsTo  uri 

```suggestion
  dcterms:conformsTo &lt;https://www.w3.org/TR/vocab-dcat/&gt; ;
```

riccardoAlbertoni committed:
remove the duplication of .org in the dcterms:conformsTo uri

riccardoAlbertoni commented:
We might want to add this addition to the change History. Perhaps, we can do it after merging #1486, so that we don't risk conflicts between these two PRs.

andrea-perego commented:
@riccardoAlbertoni said:

> We might want to add this addition to the change History. Perhaps, we can do it after merging #1486, so that we don't risk conflicts between these two PRs.

Agreed. Let's keep this on hold until #1486 is merged.

andrea-perego committed:
Remove duplication of .org in DCAT URI

andrea-perego committed:
Add relevant files to examples folder

3 files:
- catalog-record-schema.ttl (master)
- catalog-record-schema.rdf
- catalog-record-schema.jsonld

andrea-perego committed:
Merge branch 'gh-pages' into dcat-ex-record-schema

andrea-perego committed:
Update changelog and run US spellchecker

Plus other editorial fixes.

andrea-perego committed:
Fixed typo

andrea-perego commented:
> @riccardoAlbertoni said:
> 
> > We might want to add this addition to the change History. Perhaps, we can do it after merging #1486, so that we don't risk conflicts between these two PRs.
> 
> Agreed. Let's keep this on hold until #1486 is merged.

I updated the changelog, and I also took the opportunity to fix some typos and editorial issues. I also run the US spellchecker.

Some of the typos concern two property definitions not in the DCAT namespace. So the fixes should be reflected also in the corresponding RDF files.

The relevant properties are:

- `dcterms:accrualPeriodicity` in [§6.6.2 Property: frequency](https://w3c.github.io/dxwg/dcat/#Property:dataset_frequency). 

  Current definition: 
  > The frequency at which dataset is published.

  New definition: 
  > The frequency at which **a** dataset is published.

- `spdx:checksumValue` in [§6.17.2 Property: checksum value](https://w3c.github.io/dxwg/dcat/#Property:checksum_checksum_value). 

  Current definition: 
  > The checksumValue property provides a lower case hexidecimal encoded digest value produced using a specific algorithm [[SPDX]].

  New definition: 
  > The checksumValue property provides a **lowercase hexadecimal** encoded digest value produced using a specific algorithm [[SPDX]]."

andrea-perego committed:
Editorial fixes

riccardoAlbertoni commented:
I guess we want to merge the changes to the HTML  pretty soon. So I would suggest dealing with the DCAT RDF  in a new PR,  or even directly in the #1482 to avoid conflict among PRs.

> > @riccardoAlbertoni said:
> > > We might want to add this addition to the change History. Perhaps, we can do it after merging #1486, so that we don't risk conflicts between these two PRs.
> > 
> > 
> > Agreed. Let's keep this on hold until #1486 is merged.
> 
> I updated the changelog, and I also took the opportunity to fix some typos and editorial issues. I also run the US spellchecker.
> 
> Some of the typos concern two property definitions not in the DCAT namespace. So the fixes should be reflected also in the corresponding RDF files.
> 
> The relevant properties are:
> 
> * `dcterms:accrualPeriodicity` in [§6.6.2 Property: frequency](https://w3c.github.io/dxwg/dcat/#Property:dataset_frequency).
>   Current definition:
>   > The frequency at which dataset is published.
>   
>   
>   New definition:
>   > The frequency at which **a** dataset is published.
> * `spdx:checksumValue` in [§6.17.2 Property: checksum value](https://w3c.github.io/dxwg/dcat/#Property:checksum_checksum_value).
>   Current definition:
>   > The checksumValue property provides a lower case hexidecimal encoded digest value produced using a specific algorithm [[SPDX]].
>   
>   
>   New definition:
>   > The checksumValue property provides a **lowercase hexadecimal** encoded digest value produced using a specific algorithm [[SPDX]]."

andrea-perego commented:
@riccardoAlbertoni scripsit:

> I guess we want to merge the changes to the HTML pretty soon. So I would suggest dealing with the DCAT RDF in a new PR, or even directly in the #1482 to avoid conflict among PRs.

Yes, this was just meant to be a heads-up. And doing the fix in the RDF might not be so straightforward, as the RDF files with external properties / classes are still those of DCAT2.

andrea-perego committed:
Update catalog-record-schema.jsonld

andrea-perego committed:
Update catalog-record-schema.rdf

andrea-perego committed:
Update catalog-record-schema.ttl

andrea-perego committed:
Editorial fix

andrea-perego committed:
Replace "item" with "resource" + other ed fixes

- Replace "item" with "resource" when referring to an instance of dcat:Resource
- Replace "item" with "record" when referring to an instance of dcat:CatalogRecord
- Replace citation key VOCAB-DCAT-20140116 with VOCAB-DCAT-1

andrea-perego commented:
I created a PR (https://github.com/w3c/dxwg/pull/1491) on top of this one to address issue https://github.com/w3c/dxwg/issues/1490

If you think that the proposal in PR https://github.com/w3c/dxwg/pull/1491 makes sense, I suggest we wait to merge this one. Otherwise, I'll create another one.

andrea-perego committed:
Update changelog + ed fixes

andrea-perego committed:
Editorial fix

andrea-perego committed:
Merge branch 'gh-pages' into dcat-ex-record-schema

andrea-perego committed:
Merge pull request #1491 from w3c/dcat-issue-1490

Fix inconsistent use of "item" and "resource"

andrea-perego committed:
Editorial fix

andrea-perego committed:
Editorial fixes

andrea-perego committed:
Editorial fixes
